apps/dex/src/layouts/Nav/Nav.tsx

```diff
@@ -151,4 +118,48 @@ const Header = () => {
   );
 };
 
+const RowanStats = () => {
+  const { data: TVL, isLoading: isLoadingTVL } = useTVLQuery();
+  const { data: rowanPrice, isLoading: isLoadingRowanPrice } =
+    useRowanPriceQuery();
+
+  const rowanStats = [
+    {
+      id: "price",
+      icon: <RowanIcon />,
+      label: (
+        <>
+          {isLoadingRowanPrice || isNaN(rowanPrice)
+            ? "..."
+            : formatNumberAsCurrency(rowanPrice)}
+          <span className="ml-3">/</span>
+        </>
+      ),
+    },
+    {
+      id: "tvl",
+      icon: <LockIcon />,
+      label: (
+        <>
+          {isLoadingTVL || isNaN(TVL) ? "..." : formatNumberAsCurrency(TVL)} TVL
```

Comment:
i first i tried to find the `isNaN` function/pkg... then I remembered we have `Number.isNaN` and `.isNaN` is a lias...classic JS

Reply:
isNaN has been a global since I first used js

Reply:
sure thing, I highlighted that it can be confusing, behind the scenes is an alias to `Number.isNaN` 😆